scripts/visualise_results.py

```diff
@@ -6,38 +6,56 @@
 from scripts.get_summary import get_results, parse_environment_variables
 
 PERCENTILES_TO_GRAPH = (50, 90, 95, 99)
+PERCENTILES_TO_PLOT = ("50th", "90th", "95th", "99th")
+
+ADDITIONAL_METRICS_TO_GRAPH = ("PDF", "Session")
 
 
 class GraphGenerationFailed(Exception):
     pass
 
 
-def plot_data(df, number_of_days_to_plot):
-    try:
-        plt.style.use("fast")
+def plot_data(dataframes, number_of_days_to_plot):
+    plt.style.use("fast")
+    fig, axes = plt.subplots(nrows=1, ncols=2, figsize=(15, 6))
 
+    for i in range(len(dataframes)):
+        plt.subplot(1, 2, i + 1)
```

Comment:
Might need a comment to make it clear what this line is doing

Reply:
Added that in 👍